script.js

```diff
@@ -66,6 +68,9 @@ const mangabox_url_list = ["manganelo.com", "mangakakalot.com"];
 // This list is for Hitomi and proxy sites
 const hitomi_url_list = ["hitomi.la"];
 
+// This list is for pastebin and its proxy sites
+const pastebin_url_list = ["pastebin.com"];
```

Comment:
I don't think adding pastebin support is necessary since it's for very specific pastes as we're using it purely as an API response cache.

Reply:
Sure. I did not know about its use case and thought its there why not add it. I will remove it in a subsequent commit

Reply:
Removed pastebin in a subsequent commit